README.md

```diff
@@ -14,13 +14,25 @@ CMake version 3.12 is required to use these libraries.
 ### Development Environment
 Project contains files to work on Windows or Linux based OS.
 
+> Note: Project contains git submodules required to build. Before building run `git submodule update --init --recursive`
+
 #### Windows
 Use PowerShell to run {projectDir}/build.ps1
 
 #### Linux
-Use bash to run {projectDir}/build-tools/build.sh
-
-Output libraries are created in /build folder
+- Install `libssl`
+  - ```sudo apt-get install libssl-dev```
+- Install `uuid`
+  - ```sudo apt-get install uuid-dev```
+
+##### steps to build
+```
+mkdir build
+build
+cmake ../
+make
+```
+> Note: use `cmake -Duse_default_uuid=ON ../` if no uuid-dev is installed in system

```

Comment:
this embeds libuuid correct?

Reply:
makes compilation works when uuid is not in the system (like in DevOps Mac env).
Didn't run tests to see if the custom impl works thou